streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImpl.java

```diff
@@ -446,7 +446,9 @@ public void checkpoint() {
         try {
             checkpointFile.write(filteredOffsets);
         } catch (final IOException e) {
-            log.warn("Failed to write offset checkpoint file to {} for global stores: {}", checkpointFile, e);
+            log.warn("Failed to write offset checkpoint file to {} for global stores: {}." +
+                " This may occur if OS cleaned the state.dir in case when it located in /tmp directory." +
```

Comment:
`it located` -> `it is located`

`in /tmp directory` -> `in the (default) /tmp/kafka-streams directory` ?